es6.js

```diff
@@ -27,24 +27,35 @@ if (typeof window !== "undefined" && window.navigator && window.document) {
 
 define([
     //>>excludeStart('excludeBabel', pragmas.excludeBabel)
-    'babel'
+    'babel',
     //>>excludeEnd('excludeBabel')
+    'module'
 ], function(
     //>>excludeStart('excludeBabel', pragmas.excludeBabel)
-    babel
+    babel,
     //>>excludeEnd('excludeBabel')
+    module
```

Comment:
Maybe we need to rename this with `_module` (or something else) to exclude futures conflicts with native modules.